authorizer-app/src/app/components/rest-source-authorization/update-rest-source-user.component.html

```diff
@@ -77,6 +77,12 @@ <h3 *ngIf="isEditing">Edit details for User</h3>
            id="isAuthorized" [value]= "restSourceUser.isAuthorized? 'Yes':'No'">
   </div>
 
+  <div class="form-group" *ngIf="isEditing">
+    <label for="hasValidToken">Has Valid Token:</label>
```

Comment:
I see, I can remove it. It is indeed an additional information on the token status. In that case we will only have Authorized state, and it will only state if the user currently has a token or not. Not if it is a valid one.